features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -91,6 +91,12 @@
 		<feature dependency="true">openhab.tp-jupnp</feature>
 	</feature>
 
+	<feature name="openhab-core-config-discovery-addon-usb" version="${project.version}">
+		<feature>openhab-core-base</feature>
+		<feature>openhab-core-config-discovery-addon</feature>
+		<bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.addon.usb/${project.version}</bundle>
```

Comment:
Add a reference to the usb4java mvn bundle here. I think that will solve your feature verification issue.

I actually don't think you need all the special code in your pom, just add usb4java and javax.usb as dependencies.

Reply:
Ok. I fixed it.